hack/generate-manifest.sh

```diff
@@ -298,7 +317,11 @@ if $KIND; then
     # this adds a small delay before running the antrea-agent process, to give the antrea-ovs
     # container enough time to set up the br-phy bridge.
     # workaround for https://github.com/vmware-tanzu/antrea/issues/801
-    $KUSTOMIZE edit add patch startAgent.yml
+    if $COVERAGE; then
+        $KUSTOMIZE edit add patch startAgentCov.yml 
```

Comment:
when I test your PR, I get the following error:
```bash
abas-a01:antrea abas$ ./hack/generate-manifest.sh --kind --coverage | more > 1
2020/09/22 17:15:27 startAgentCov.yml has no match
```

which makes sense to me because you assume that file `build/yamls/patches/kind/startAgentCov.yml` exists while it does not. I see the same error when looking at Kind CI logs. However we still get coverage data because we previously applied `build/yamls/patches/coverage/startAgentCov.yml`, which also happens to work in Kind.

I think you should either add the missing file, or change this code to:
```bash
if ! $COVERAGE; then
    $KUSTOMIZE edit add patch startAgent.yml
fi
```
but you will also need to add a comment to explain what's going on

Reply:
maybe we can copy the startAgentCov.yml here before add the patch if coverage enabled?

I added the below before "$KUSTOMIZE edit add patch startAgentCov.yml" and tested on local there is no error anymore.  I think it is OK if we add startAgentCov.yml patch twice.
cp ../../patches/coverage/startAgentCov.yml .

Reply:
works for me, thanks